kubernetes_platform/proto/kubernetes_executor_config.proto

```diff
@@ -34,6 +34,8 @@ message KubernetesExecutorConfig {
     int64 active_deadline_seconds = 10;
     repeated FieldPathAsEnv field_path_as_env = 11;
     repeated Toleration tolerations = 12;
+    repeated NodeAffinity node_affinities = 13;

```

Comment:
```suggestion
    repeated NodeAffinity node_affinity = 13;
```

In Kubernetes spec, the node affinity key name is `nodeAffinity`. We should use plural name such as `tolerations` if it's defined the same way in Kubernetes spec.
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#nodeaffinity-v1-core

Reply:
Fair enough. One thing, I am not following the Kubernetes spec, but intend to generate the `spec.affinity` section using the data from the list of NodeAffinity/PodAffinity object. 
If I should follow the k8s api spec more closely, let me know and will rethink how to set this up

Reply:
You don't have to follow the Kubernetes api spec. I'm just thinking using the `node_affinity` name might make more sense in terms of SEO. The only other place I saw people use `node_affinities` is in GCP compute instance.

Reply:
Fair enough. Seems like all references are singular, indeed. Also from the definition in the spec:
> Affinity is a group of affinity scheduling rules.
So I've renamed the field in the config to pod_affinity and node_affinity as you have suggested. And the classnames to `NodeAffinityRule` and `PodAffinityRule`. Then `pod_affinity` is a list of `PodAffinityRule`s and `node_affinity` is a list of `NodeAffinityRule`s being consistent with the semantics (a group of rules)